src/backend/distributed/sql/citus--11.0-4--11.1-1.sql

```diff
@@ -79,3 +79,59 @@ DROP FUNCTION pg_catalog.get_all_active_transactions(OUT datid oid, OUT process_
 
 DROP FUNCTION pg_catalog.isolate_tenant_to_new_shard(table_name regclass, tenant_id "any", cascade_option text);
 #include "udfs/isolate_tenant_to_new_shard/11.1-1.sql"
+
+CREATE TYPE citus.citus_job_status AS ENUM ('scheduled', 'running', 'finished', 'cancelled', 'failing', 'failed');
+ALTER TYPE citus.citus_job_status SET SCHEMA pg_catalog;
+
+CREATE TABLE citus.pg_dist_background_jobs (
+    job_id bigserial NOT NULL,
+    state pg_catalog.citus_job_status DEFAULT 'scheduled' NOT NULL,
+    job_type name,
+    description text,
+    started_at timestamptz,
+    finished_at timestamptz,
+
+    CONSTRAINT pg_dist_background_jobs_pkey PRIMARY KEY (job_id)
+);
+ALTER TABLE citus.pg_dist_background_jobs SET SCHEMA pg_catalog;
+GRANT SELECT ON pg_catalog.pg_dist_background_jobs TO PUBLIC;
+GRANT SELECT ON pg_catalog.pg_dist_background_jobs_job_id_seq TO PUBLIC;
+
+CREATE TYPE citus.citus_task_status AS ENUM ('blocked', 'runnable', 'running', 'done', 'error', 'unscheduled', 'cancelled');
+ALTER TYPE citus.citus_task_status SET SCHEMA pg_catalog;
+
+CREATE TABLE citus.pg_dist_background_tasks(
+    job_id bigint NOT NULL REFERENCES pg_catalog.pg_dist_background_jobs(job_id),
+    task_id bigserial NOT NULL,
+    owner regrole NOT NULL DEFAULT CURRENT_USER::regrole,
+    pid integer,
+    status pg_catalog.citus_task_status default 'runnable' NOT NULL,
+    command text NOT NULL,
+    retry_count integer,
+    not_before timestamptz, -- can be null to indicate no delay for start of the task, will be set on failure to delay retries
+    message text,
```

Comment:
```suggestion
    message text NOT NULL,
```

Reply:
messages are actually `NULL` till their is either an error or a finished task.
What would the benefit of `NOT NULL` be here?

Reply:
mainly not having to check for NULL when trying to print it